packages/core/src/eth/wallet.ts

```diff
@@ -187,6 +188,19 @@ export abstract class EthLikeWallet extends Wallet<EthTransaction> {
     };
   }
 
+  async resendTransaction(
+    request: Omit<ResendTransactionRequest, "walletId">
```

Comment:
In wallet API, `ResendTransactionRequest` has `walletId`. You may have to check that parameter to Wallet Core Team.